source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
                      <p>Watermark</p>
                      <p>Stamp</p>
                      <p>physMedium</p>-->
+                     <p>Encoding the extent and dimensions of a source:</p>
+                     <p><specList>
+                        <specDesc key="extent"/>
+                     </specList></p>
```

Comment:
Please check indentation and line breaks of all the blocks with

```
<p><specList>
 ...
</specList></p> 
```

Should be:


```
<p>
    <specList>
 ...
    </specList>
</p> 
```